packages/@o3r/core/schematics/module/templates/base/package.json.template

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.0-placeholder",
+  "description": "<%= description || ('Otter module ' + name) %>",
+  "keywords": [
+    "apis-manager",
+    "otter"
+  ],
+  "scripts": {
+    "nx": "<%= isNxContext ? 'nx' : 'ng' %>",
+    "ng": "<%= runner %> nx",
+    "build": "<%= runner %> nx build apis-manager",
+    "prepare:build:builders": "<%= runner %> cpy 'collection.json' dist && <%= runner %> cpy 'schematics/**/*.json' dist/schematics",
+    "build:builders": "tsc -b tsconfig.builders.json --pretty"
+  },
+  "peerDependencies": {
+    "@angular/common": "<%= versions['angular'] %>",
+    "@angular/core": "<%= versions['angular'] %>",
+    "@o3r/dev-tools": "<%= otterVersion %>",
+    "@o3r/schematics": "<%= otterVersion %>"
+  },
+  "peerDependenciesMeta": {
+    "@angular-devkit/schematics": {
+      "optional": true
+    },
+    "@o3r/schematics": {
+      "optional": true
+    },
+    "@schematics/angular": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "tslib": "^2.4.1"
```

Comment:
could be taken from generatorDependencies ?

Reply:
From `dependecies` actually but yes, I forgot this one, thanks ;)